src/components/games/GameControlsMod.svelte

```diff
@@ -240,9 +303,9 @@
             openREPLForMod(getInternalName(activeGame), modName, modSource);
           }}>{$_("gameControls_button_openREPL")}</DropdownItem
         >
-      {/if}
-      <DropdownDivider />
-      <DropdownItem
+        {/if}
+        <DropdownDivider />
+        <DropdownItem
```

Comment:
if you're going to make this change, can you make it to the `GameControls.svelte` file as well?

Reply:
What change are you referring to here? Seems like these lines are just indentation changes (that I'll revert)